nautobot/extras/models/jobs.py

```diff
@@ -126,6 +126,7 @@ class Job(PrimaryModel):
         help_text="Whether the Python module and class providing this job are presently installed and loadable",
     )
     enabled = models.BooleanField(default=False, help_text="Whether this job can be executed by users")
+    is_job_hook_receiver = models.BooleanField(default=False)
```

Comment:
Is the intent that this be a "hidden" model field, not exposed in the UI or API or editable by a power user? Or does this still need to be added to the serializer, detail-view template, edit-form, etc.?

Reply:
I think this should be visible via API or UI (when looking at the Job Model) but should not be editable since it's a tight dependency on the class the Job inherits from.